### PR TITLE
GOAT is a bad choice for simple diagrams

### PR DESCRIPTION
**Pull request by martinthomson**

### CODE REVIEW

**Comment by martinthomson** [comment]
I might also add [ditaa](http://ditaa.sourceforge.net/) and [svgbob](https://ivanceras.github.io/svgbob-editor/), but those aren't as well-integrated.

**Comment by kesara** [comment]
Currently, author-tools only support `aasvg`.
Author tools did have support for `GoAT` in the past, but support was removed due to toolchain issues.

File: diagrams.md [hunk 1]
```diff
@@ -69,22 +69,14 @@ Dia is simple to use. Save your drawing as xxx.dia, then export it as xxx.svg.
...
-### Community tips
-#### Julian Reschke, October 2021
-(Extract from his [list email](https://mailarchive.ietf.org/arch/msg/rfc-interest/4wlkqfxZIFFPnvhHsmVwxthv_fE/))
-
-So the problem is that goat positions every non-line-art character
-separately.
-
-The [attached script](https://mailarchive.ietf.org/arch/msg/rfc-interest/4wlkqfxZIFFPnvhHsmVwxthv_fE/2/) will try to recombine SVG text elements that are
-"next" to each other (as produced by Goat, not in general!)
+This tool is a replacement for [goat](https://github.com/blampe/goat), which scrambles the placement of text in diagrams.
```
Comment (martinthomson) [inline diagrams.md:77]: Jean's comment as a suggestion:

```suggestion
This tool is inspired by [goat](https://github.com/blampe/goat) and uses a modified version of the original [markdeep](https://casual-effects.com/markdeep/) code.
```

**Comment by martinthomson** [comment]
Yeah, good point.  It is, after all, just a bug.

**Pull request merged**